gcc/rust/hir/tree/rust-hir-item.h

```diff
@@ -3144,6 +3150,39 @@ class ExternalFunctionItem : public ExternalItem
   }
 };
 
+class ExternalTypeItem : public ExternalItem
+{
+public:
+  ExternalTypeItem (Analysis::NodeMapping mappings, Identifier item_name,
+		    location_t locus)
+    : ExternalItem (std::move (mappings), std::move (item_name),
+		    Visibility (Visibility::PRIVATE),
```

Comment:
Extern types may accept pub visibility modifiers.

Reply:
I'll take care of this in a later PR, I think it'll be easier